third_party/gRPC_utils/proto_reflection_descriptor_database.cc

```diff
@@ -32,6 +32,7 @@ using grpc::reflection::v1alpha::ServerReflection;
 using grpc::reflection::v1alpha::ServerReflectionRequest;
 using grpc::reflection::v1alpha::ServerReflectionResponse;
 
+const uint8_t g_timeoutGrpcMainStreamSeconds = 20; //using default gwhisper timeout of 20 seconds.
```

Comment:
increased since we already have  default rpc timeout of 30secs anyway, is that too big? we saw on slower systems 10secs timeout quite often.